src/assets/scripts/client/airport/AirspaceModel.js

```diff
@@ -102,34 +113,46 @@ export default class AirspaceModel extends BaseModel {
         this.airspace_class = '';
     }
 
+    transformPoly() {
```

Comment:
missing docblock.  I'd really love a description of this one, what is this and why is it important?

Reply:
added docblock. The description could be better though